### ISSUE TITLE
My number of epochs is much higher than those in the provided loss graphs.

### ISSUE THREAD

**Issue by rfitz123** [taki0112/StyleGAN-Tensorflow#11]
I was comparing my losses to the graph of the losses from the owner of the repo and noticed that my training is running about 8x the number of epochs. My images are 512x512 and I have 1800 or so images, but my iteration is left to the default 120, so I am confused why I have so many more epochs than the original training. However, the epochs was the same when I had 900 images, which I believe is the same as the original. Any ideas why mine is running so much more epochs? It takes 1 week to train on my RTX 2070.

Edit: I am now realizing it may be because my gpu_num is 1, but is it really possible the repo owner used 8 GPUs? I find that unlikely.
